src/bezpath.rs

```diff
@@ -1245,7 +1269,7 @@ mod tests {
     }
 
     #[test]
-    #[should_panic(expected = "Can't start a segment on a ClosePath")]
+    #[should_panic(expected = "no open subpath")]
```

Comment:
These tests will fail with `cargo test --release`. Not sure what the best approach is, here?

Reply:
> One additional thought: there are various other ways to construct a bezpath, such as by pushing elements directly, or via `FromIterator`. Should we validate those as well?

It's a good question. Probably yes, but in some cases (like extending from a slice) it might be expensive. I think the current state is very clear documentation about what's correct, which is the immediate problem.

Reply:
Regarding `test --release` I'm not sure what's considered best practice. I would prefer the checks not be in release code though, as it might be a performance critical path.

Reply:
yes, I agree with not having the tests in release mode. Can we disable these tests in release?